modules/playing-card/widgets/playing-card-widget.php

```diff
@@ -0,0 +1,274 @@
+
+		$repeater->start_controls_tab(
+			'content',
+			[
+				'label' => esc_html__( 'Content', 'elementor' ),
+			]
+		);
+
+		$repeater->add_control(
+			'card_number',
+			[
+				'label' => esc_html__( 'Card Number', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'default' => 'A',
+				'options' => [
+					'A' => 'A',
+					'2' => '2',
+					'3' => '3',
+					'4' => '4',
+					'5' => '5',
+					'6' => '6',
+					'7' => '7',
+					'8' => '8',
+					'9' => '9',
+					'10' => '10',
+					'J' => 'J',
+					'Q' => 'Q',
+					'K' => 'K',
+				],
+			]
+		);
+
+		$suite_options = array_map( function ( $suite ) {
+			return $suite['name'];
+		}, $this->suites_map );
+
+		$repeater->add_control(
+			'card_suit',
+			[
+				'label' => esc_html__( 'Card Suit', 'elementor' ),
+				'type' => Controls_Manager::SELECT,
+				'default' => '❤️',
+				'options' => $suite_options,
+			]
+		);
+
+		$repeater->end_controls_tab();
+
+		$repeater->start_controls_tab(
+			'style',
+			[
+				'label' => esc_html__( 'Style', 'elementor' ),
+			]
+		);
+
+		$repeater->add_responsive_control(
+			'card_padding',
+			[
+				'label' => esc_html__( 'Padding', 'elementor' ),
+				'type' => Controls_Manager::DIMENSIONS,
+				'size_units' => [ 'px', '%', 'em', 'rem', 'vw', 'custom' ],
+				'selectors' => [
+					'{{WRAPPER}} {{CURRENT_ITEM}}' => 'padding: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
+				],
+			]
+		);
+
+		$repeater->end_controls_tab();
+
+		$this->add_control(
+			'playing_cards_list',
+			[
+				'label' => esc_html__( 'Playing cards', 'elementor' ),
+				'type' => Controls_Manager::REPEATER,
+				'fields' => $repeater->get_controls(),
+				'default' => [
+					[
+						'playing_card' => [
+							'card_number' => 'A',
+							'card_suit' => '❤️',
+						],
+					],
+				],
+			]
+		);
+
+		$this->end_controls_section();
+	}
+
+	private function add_playing_cards_style_section() {
+		$this->start_controls_section(
+			'section_playing_card_style',
+			[
+				'label' => esc_html__( 'Playing Cards', 'elementor' ),
+				'tab' => Controls_Manager::TAB_STYLE,
+			]
+		);
+
+		$this->add_responsive_control(
+			'playing_card_item_space_between',
+			[
+				'label' => esc_html__( 'Space between Items', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px' ],
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 200,
+					],
+				],
+				'default' => [
+					'size' => 0,
+				],
+				'selectors' => [
+					'{{WRAPPER}} > .elementor-widget-container > .e-playing-cards-wrapper' => 'gap: {{SIZE}}{{UNIT}}',
+				],
+			]
+		);
+
+		$this->end_controls_section();
+	}
+
+	/**
+	 * Render playing card widget output on the frontend.
+	 *
+	 * Written in PHP and used to generate the final HTML.
+	 *
+	 * @since 1.0.0
+	 * @access protected
+	 */
+	protected function render() {
+		$settings = $this->get_settings_for_display();
+
+		?>
+		<div class="e-playing-cards-wrapper">
```

Comment:
Try to keep your class names short.

```suggestion
		<div class="e-playing-cards">
```